GETTING_STARTED.md

```diff
@@ -43,6 +24,27 @@ Fetch the latest version of `master` from `upstream` (ie. the main repo):
 
     git fetch upstream master
 
+### Installation
+
+This project needs specific ruby/bundler versions as well as node/yarn specific versions. For a local setup you will need:
+
+* Install or change your Ruby version according to the one specified at [.ruby-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.ruby-version) file.
+    - To manage versions, it's recommended to use [rbenv](https://github.com/rbenv/rbenv).
+* Install [nodenv](https://github.com/nodenv/nodenv) to ensure the correct [.node-version](https://github.com/openfoodfoundation/openfoodnetwork/blob/master/.node-version) is used.
+    - [nodevn](https://github.com/nodenv/nodenv) is recommended as a node version manager.
+* PostgreSQL database
+* Redis (for background jobs)
+* Chrome (for testing)
+
+The following guides will provide OS-specific step-by-step instructions to get these requirements installed:
+- [Ubuntu Setup Guide][ubuntu]
+- [Debian Setup Guide][debian]
+- [OSX Setup Guide][osx]
+
+For those new to Rails, the following tutorial will help get you up to speed with configuring a [Rails environment](http://guides.rubyonrails.org/getting_started.html).
+
+Another way to make it work locally would be using Docker. See the [Docker setup guide](docker/README.md). You will need to setup git and others aditional steps.
```

Comment:
```suggestion
Another way to make it work locally would be using Docker. See the [Docker setup guide](docker/README.md).
```

I think that we can take Git for granted since the code is hosted on Github. And the Docker readme should contain all needed steps without us needing to mention it here. There was also a little typo in that sentence.